### ISSUE TITLE
`no-invalid-position-at-import-rule` false positive with `@use`

### ISSUE THREAD

**Issue by boris-petrov** [stylelint/stylelint#5322]
```scss
@use 'sass:math';

@import 'bootstrap';
```

This leads to a `no-invalid-position-at-import-rule` warning which is wrong because I can't put `@use` after the `@import` statement.

> Which rule, if any, is the bug related to?

`no-invalid-position-at-import-rule`

> What code is needed to reproduce the bug?

See above.

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "no-invalid-position-at-import-rule": true
  }
}
```

> Which version of stylelint are you using?

`13.13.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, SCSS.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

There is a warning.

This issue is similar to https://github.com/stylelint/stylelint/issues/5263.

**Comment by hudochenkov** [comment]
This rule is for CSS. You can disable it for Sass: https://github.com/stylelint/stylelint/issues/5133#issuecomment-834466368
